docs/manual/scala/guide/build/MultipleBuilds.md

```diff
@@ -40,6 +40,9 @@ The `hello` Lagom service can be imported by adding the following declaration to
 
 The first argument passed to `lagomExternalScaladslProject` is the name that will be used in your build to refer to this externally defined project. While, the second argument provides the dependency to the `hello-impl` JAR, using the conventional sbt syntax for declaring dependencies. Note in fact that the `lagomExternalScaladslProject` method returns a sbt `Project`, which you can further customize if needed.
 
+You can further configure the service (what ports it is available on, the address it is bound to, etc...) using [[the same settings than a managed Lagom Service|ConfiguringServicesInDevelopment]].
```

Comment:
not native speaker, but I think the right working is "the same settings as..." instead of 'than'

Reply:
🤷‍♂️

Reply:
TIL: https://forum.wordreference.com/threads/same-than-same-as-same-that.400069/ (you are right)